### ISSUE TITLE
Elegant way of handling *both* raw data (A_data, B_data) and summarized data (A_counts, A_exposure, etc...)

### ISSUE THREAD

**Issue by FrankPortman** [FrankPortman/bayesAB#4]
Next release I want to have both the current setup (input A_data, B_data directly) and summarized data (A_sum, A_length, B_sum, B_length). This should not complicate the architecture if done correctly.

**Comment by benaddict** [comment]
@FrankPortman looking for a way to use summary data instead of raw data for metrics that we know are following a normal distribution. The above seems to not work in the current version?

**Comment by FrankPortman** [comment]
You're right that this didn't actually make it in to the current version. I would be open to reviewing PRs that support this otherwise it's pretty far in my backlog unfortunately. I can give you some pointers on where I'd start if you are interested in adding it.
